bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/TeslaChannelSelectorProxy.java

```diff
@@ -366,18 +366,6 @@ public State getState(String s, TeslaChannelSelectorProxy proxy, Map<String, Str
                 return super.getState(s);
             }
         },
-        FORCE_REFRESH(null, "forcerefresh", OnOffType.class, false) {
```

Comment:
@kaikreuzer had suggested to keep this channel as is to ensure backwards compatibility